doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-run-test-proc.adoc

```diff
@@ -4,7 +4,7 @@
 After creating a test scenario template and defining the test scenarios, you can run the tests to validate your business rules and data.
 
 .Procedure
-. At the top of the test scenarios designer, click *Run Test* image:AuthoringAssets/test-scenarios-run-test-icon.png[Run Test icon] icon.
+. At the top of the test scenarios designer, click *Run Test* image:AuthoringAssets/test-scenarios-run-test-icon.png[Run Test icon] icon to execute all the test scenarios from the table at once. To run a single test scenario, right click on the row that you want to execute and select *Run scenario*.
```

Comment:
"Execute" or "run"?

Reply:
"Run" is preferred.
These steps are already updated in the BXMSDOC-4115 JIRA.